CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+1.5.3 Release notes (2021-11-12)
+=============================================================
+
+### Changes
+
+* Removed addRingPublishingGDPRListener and removeRingPublishingGDPRListener and replace by setRingPublishingGDPRListener
```

Comment:
wypadało by to pewnie zaktualizować w dokumentacji na ringpublishing

Reply:
@grzegorzmalopolski  pomozesz mi w tym ?